components/victron/victron.cpp

```diff
@@ -975,6 +979,11 @@ void VictronComponent::handle_value_() {
     return;
   }
 
+  if (label_ == "HC#") {
```

Comment:
Where can I find this TAG `HC#`
I don't see it in [datasheet ver 3.33 ](https://www.victronenergy.com/upload/documents/VE.Direct-Protocol-3.33.pdf)

Reply:
Based upon communications here:
https://community.victronenergy.com/questions/226131/bluesmart-ip22-charger-and-vedirect.html
`HC#` is the Hardware Revision. 
I'd expect it to be similar to the firmware version?